Update forced-tunneling.md

adstuart opened the pull request:
Clarify that azure firewall in VWAN is always deployed in forced tunnel mode, this is not a toggle

adstuart committed:
Update forced-tunneling.md

Clarify that azure firewall in VWAN is always deployed in forced tunnel mode, this is not a toggle

vhorne commented on articles/firewall/forced-tunneling.md:
```diff
@@ -16,7 +16,7 @@ When you configure a new Azure Firewall, you can route all Internet-bound traffi
 
 > [!IMPORTANT]
-> If you deploy a Secured Virtual Hub in forced tunnel mode, advertising the default route over Express Route or VPN Gateway is not currently supported. A fix is being investigated.
+> If you deploy Azure Firewall inside of a Virtual WAN Hub (Secured Virtual Hub), please note that advertising the default route over Express Route or VPN Gateway is not currently supported. A fix is being investigated.
```
```suggestion
> If you deploy Azure Firewall inside of a Virtual WAN Hub (Secured Virtual Hub), advertising the default route over Express Route or VPN Gateway is not currently supported. A fix is being investigated.
```

vhorne replied:
@adstuart additional wording not needed.

adstuart committed:
Update articles/firewall/forced-tunneling.md

yes the important point is to capture that you do not have the choice to deploy or not deploy in forced tunnel mode, when putting azfw in vwan

Co-authored-by: Vic <15256338+vhorne@users.noreply.github.com>
